src/core_memusage.h

```diff
@@ -26,27 +26,49 @@ static inline size_t RecursiveDynamicUsage(const CTxOut& out) {
     return RecursiveDynamicUsage(out.scriptPubKey);
 }
 
-//TODO Account for CTxWitness.
+static inline size_t RecursiveDynamicUsage(const CTxInWitness& wit) {
```

Comment:
all these ``static`` are redundant, I copied them for symmetry, though they should probably be removed.